test/unit/org/apache/cassandra/tools/OfflineToolUtils.java

```diff
@@ -79,6 +79,8 @@ public abstract class OfflineToolUtils
     "Attach Listener", // spawned in intellij IDEA
     "JNA Cleaner",     // spawned by JNA
     "ThreadLocalMetrics-Cleaner", // spawned by org.apache.cassandra.metrics.ThreadLocalMetrics
+    "Native reference cleanup thread",
+    "^ForkJoinPool\\.commonPool-worker-\\d+$"
```

Comment:
do we need this change? (I haven't noticed fork-join/parallel streams related logic in the changes..)

Reply:
I had some local failures due to these threads getting picked up